spec/util/createErrorClass-spec.ts

```diff
@@ -0,0 +1,28 @@
+/** @prettier */
+import { createErrorClass } from 'rxjs/internal/util/createErrorClass';
+import { expect } from 'chai';
+
+describe('createErrorClass', () => {
+  it('should create a class that subclasses error and has the right properties', () => {
+    const MySpecialError: any = createErrorClass(
+      (_super) =>
+        function MySpecialError(this: any, arg1: number, arg2: string) {
+          _super(this);
```

Comment:
tiny nit: would it be cleaner to call `_super(this, 'Super special error')` instead?

Reply:
_super is probably a bad name, I'm just not sure what else to call it. :\